gobblin-core/src/main/java/gobblin/writer/AvroHdfsTimePartitionedWriter.java

```diff
@@ -172,9 +170,11 @@ public void write(GenericRecord record) throws IOException {
     // If the path is in pathToWriterMap simply retrieve the writer, and write the record
     if (!this.pathToWriterMap.containsKey(writerOutputPath)) {
 
-      LOG.info("Creating a new DataWriter for path: " + new Path(this.baseFilePath, writerOutputPath));
+      Path writerOutputPathWithBase = new Path(this.baseFilePath, writerOutputPath);
+
+      LOG.info("Creating a new DataWriter for path: " + writerOutputPathWithBase);
```

Comment:
Alright, sounds fine. baseFilePath is based off of the configuration key WRITER_FILE_PATH, though, are you sure we want to rename it?